nemo/core/__init__.py

```diff
@@ -12,5 +12,5 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-import nemo.core.classes as classes
-import nemo.core.neural_types as neural_types
+from nemo.core.classes import *
+from nemo.core.neural_types import *
```

Comment:
let's keep namespace for neural-types related stuff at: "nemo.core.neural_types.*"  - it has many element-type classes that would pollute "nemo.core.*"

Reply:
Updated this so now we have:
```
>>> import nemo.core
>>> nemo.core.
nemo.core.Dataset(        nemo.core.NeuralModule(   nemo.core.common          nemo.core.nemo
nemo.core.FileIO(         nemo.core.Serialization(  nemo.core.dataset         nemo.core.neural_types
nemo.core.Model(          nemo.core.Typing(         nemo.core.modelPT         nemo.core.typecheck(
nemo.core.ModelPT(        nemo.core.classes         nemo.core.module
```